modules/pico_tftp.c

```diff
@@ -13,8 +13,11 @@
 #include <pico_tftp.h>
 #include <pico_strings.h>
 
-#define tftp_dbg(...) do {} while (0)
-//#define tftp_dbg dbg
+#ifdef DEBUG_TFTP
+#define tftp_dbg dbg
+#else
+#define tftp_dbg(...) do {} while(0) /* so_mcast_dbg in pico_socket.c */
```

Comment:
is the comment supposed to be there? Looks like a copy-paste error, and happens in a few other places as well.